OPHD/UI/PopulationPanel.cpp

```diff
@@ -126,19 +130,41 @@ void PopulationPanel::update()
 
 	position.y += fontBoldHeight + constants::Margin;
 	const auto textOffset = NAS2D::Vector{IconSize + constants::Margin, (IconSize / 2) - (fontHeight / 2)};
+	const int populationStatisticsWidth = mFont.width(constants::PopulationBreakdown) + constants::Margin;
```

Comment:
Very minor point here, though I was thinking this line doesn't really relate to the surrounding code. Printing of the mentioned strings happens before the block of code above, and use of this value happens below the block of code below. It seems like this line should be moved to one of those two places, probably after the block below, where it's used.

Or potentially, the value could be inlined, since it's only used once. Though that might make the line where it's used a bit long.

Reply:
Inlining made it no longer than the surrounding code, its intent appears obvious. So Ill inline it.